dapr/proto/runtime/v1/dapr.proto

```diff
@@ -92,6 +92,9 @@ service Dapr {
   // SubscribeConfiguration gets configuration from configuration store and subscribe the updates event by grpc stream
   rpc SubscribeConfigurationAlpha1(SubscribeConfigurationRequest) returns (stream SubscribeConfigurationResponse) {}
 
+  // UnSubscribeConfiguration unsubscribe the subscription of configuration
+  rpc UnSubscribeConfigurationAlpha1(UnSubscribeConfigurationRequest) returns (UnSubscribeConfigurationResponse) {}
```

Comment:
nit: `UnSubscribeConfigurationAlpha1` should be `UnsubscribeConfigurationAlpha1`. Same for other references of UnSubscribe.

Reply:
Thanks, I would fix it.